mission_manager/src/mission.cpp

```diff
@@ -78,12 +82,15 @@ void Mission::Stop() {
     // Clean up all of the behavior publishers and messages
     ROS_INFO("Stopping and cleaning up active behaviors");
     Behavior* cur_behavior = getNextBehavior(true);
-    do {
+    do
+    {
       cur_behavior->stopBehavior();
       cur_behavior = getNextBehavior();
-
-    } while (cur_behavior != NULL);
-  } else if (GetState() == MissionState::ABORTING) {
+    }
+    while (cur_behavior != NULL);
```

Comment:
@patapufate nit: can't this `while` statement be right next to the closing curly brace ?

Reply:
No. Roslint requires "} should be on a line by itself"

Reply:
I see. Thanks for the explanation.